source/MaterialXGenShader/ShaderGraph.cpp

```diff
@@ -1347,7 +1347,8 @@ string ShaderGraph::populateColorTransformMap(ColorManagementSystemPtr colorMana
     }
 
     const string& sourceColorSpace = input->getActiveColorSpace();
-    if (shaderPort && !sourceColorSpace.empty())
+    // Note that "none" means no transform should be applied.
+    if (shaderPort && !sourceColorSpace.empty() && sourceColorSpace != "none")
```

Comment:
This is a good proposal for the future, but I'd recommend keeping this separate from the `convert` node improvement, since this change has the potential to alter the visuals associated with existing materials.

Additionally, based on conversations with Naty Hoffman and other experts in this area, we may end up removing the "none" markings from pseudo-colors in PBR nodes, and that change ought to be made before activating the render-time effect of the marking.